components/textfield/index.css

```diff
@@ -97,8 +97,8 @@ governing permissions and limitations under the License.
 	--spectrum-textfield-icon-spacing-inline-end-override: 32px; /* TODO - placeholder - reevaluate after combobox migration */
 
 	/* TODO - icon spacing override for search - reevaluate after search is migrated */
-	--spectrum-Textfield-workflow-icon-width: 18px;
-	--spectrum-Textfield-workflow-icon-gap: 6px;
+	--spectrum-textfield-workflow-icon-width: 18px;
+	--spectrum-textfield-workflow-icon-gap: 6px;
```

Comment:
This is a good catch, but I wanted to call out that it's a change that we'll need to be aware of when I merge.

Reply:
What if we threw the misspelled variable into the varstack as a fallback with a deprecation note?

Reply:
@mlogsdon18 would you mind making this change? Having it as a fallback would mean that we wouldn't have to selectively do a breaking change for this component.